src/tests/test_utils/common_test_utils/include/common_test_utils/ov_test_utils.hpp

```diff
@@ -71,3 +71,13 @@ std::shared_ptr<ov::op::v0::Constant> create_constant(const std::vector<T>& data
 }
 
 std::shared_ptr<ov::op::v0::Constant> create_zero_constant(const ov::element::Type_t& et, const ov::Shape& shape);
+
+namespace ov {
+namespace test {
+namespace utils {
+
+bool is_tensor_iterator_exist(const std::shared_ptr<ov::Model>&);
```

Comment:
```suggestion
bool is_tensor_iterator_exist(const std::shared_ptr<ov::Model>& model);
```
Is required to pass reference to shared pointer maybe use just ov::Model

```suggestion
bool is_tensor_iterator_exist(const ov::Model& model);
```

Reply:
Usually we operate with std::shared_ptr<ov::Model> not with ov::Model. So it is just more convenient to use this function with shared_ptr